serialization/build.gradle

```diff
@@ -2,6 +2,8 @@ plugins {
     id 'corda-api.common-library'
     id 'corda.common-publishing'
     id 'corda.java-only'
+    id 'org.jetbrains.dokka'
+    id 'kotlin'
```

Comment:
@chrisr3  thoughts on this ?  - I want to insure no other impacts. 

Context Java docs generated by this module are not styled in the dokka format, which means our docs website had incorrect rendering for javadoc related to this module. -Docs team has raised this as a issue which needs resolving prior to beta. 

It seems however Dokka module will _only_  work on a java only module when kotlin plugin is also applied even if not used. 

see [here ](https://github.com/Kotlin/dokka/issues/2421)

Reply:
I was hoping to exclude the possibility of anyone adding more Kotlin code to that module. In fact, didn't I create a `corda.java-only` plugin fo prevent people apply the Kotlin plugin again?

I also thought I'd ported the KDoc to Javadoc.